### ISSUE TITLE
ALAN WAKE 2

### ISSUE THREAD

**Issue by GUTS2024** [Nukem9/dlssg-to-fsr3#32]
Has anyone made it work with the mod?

**Comment by Vladzor87do** [comment]
its actually working, just try it yourself

**Comment by Asso83** [comment]
The file to replace dbghelp.dll i didn’t find it in AW2 (epic games) , did anyone find it ?

**Comment by xatornet** [comment]
> It doesn't work, I can even activate FG, but when playing the campaign it crashes
> 
> > está realmente funcionando, tente você mesmo
> 
> It doesn't work, I can even activate FG, but when playing the campaign it crashes

Yep, as soon as it tries to load the savegame, It gives me this:

![image](https://github.com/Nukem9/dlssg-to-fsr3/assets/43992639/46047167-6af3-4583-a97d-9ad1918a8b5b)

**Comment by nanowell** [comment]
Crashed as soon game render starts

**Comment by Jackich2** [comment]
tried 0.41 and the game crash  wait more)

**Comment by darthmalak1986** [comment]
keeps crashing on 0.41

**Comment by ita360** [comment]
at me when enable FG 

ID3D12Resource::Map: due to removed device

Fence value is UINT64_MAX (device removed): due to removed device

CreateCommittedResource: due to removed device

**Comment by xatornet** [comment]
Update 0.5 it's working for me now. Great work!

![image](https://github.com/Nukem9/dlssg-to-fsr3/assets/43992639/9a09a9ea-3b7a-4521-b710-5a20be908bda)
